source/base/standard_tensors.inst.in

```diff
@@ -0,0 +1,27 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2016 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE at
+// the top level of the deal.II distribution.
+//
+// ---------------------------------------------------------------------
+
+
+for (deal_II_dimension : DIMENSIONS; number : REAL_SCALARS)
+{
+    template
+    class StandardTensors<deal_II_dimension,number>;
+}
+
+//for (deal_II_dimension : DIMENSIONS; number : COMPLEX_SCALARS)
```

Comment:
Didn't compile :-( I'll just remove this, because I don't see application for it at the moment.